app/views/partials/card/payments/in_credit.scala.html

```diff
@@ -20,9 +20,22 @@
 
 @(amount: BigDecimal, appConfig: FrontendAppConfig)(implicit request: AuthenticatedRequest[_], messages: Messages)
 
+@statementLink = @{
+ Some(appConfig.getPortalUrl("balance")(request.ctEnrolment)).map { url => url }
```

Comment:
I don't see why this needs to be lifted into a some?
this applies to the other templates

Reply:
Done